src/Symfony/Component/Workflow/Tests/WorkflowBuilderTrait.php

```diff
@@ -19,6 +19,7 @@ private function createComplexWorkflowDefinition()
         $transitions[] = $transitionWithMetadataDumpStyle;
         $transitions[] = new Transition('t4', 'd', 'f');
         $transitions[] = new Transition('t5', 'e', 'g');
+        $transitions[] = new Transition('t3', 'f', 'e');
```

Comment:
As you state, IMHO, it's a bad idea to change theses fixtures :) You don't need it
Could you revert theses changes?

Reply:
done !